dvc/hash_info.py

```diff
@@ -11,6 +11,7 @@
 class HashInfo:
     PARAM_SIZE = "size"
     PARAM_NFILES = "nfiles"
+    PARAMS_TO_IGNORE = ("isexec",)
```

Comment:
@efiop I'm not sure if this is a good approach and there could be a better way to ignore `isexec` from .dvc file in `HashInfo`. I plan to look at a better way to handle it later but maybe you have some quick suggestions?

Reply:
@dudarev Take a look at `loadd_from` in `dvc/output/__init__.py`. You just need to pop the value for isexec from info, same as we do with things like `desc`.

Reply:
Thanks! I've modified the code to pop `isexec` in `loadd_from`.